src/middlewared/middlewared/plugins/enclosure_/enclosure2.py

```diff
@@ -1,3 +1,8 @@
+# Copyright (c) 2023 - iXsystems Inc.
```

Comment:
Is it possible to add copyright to `LICENSE.IX` so we just, in theory, have to update it in 1 place instead of everywhere 😄 ?

Reply:
Not a bad idea ;) I'll do that

Reply:
Done, thanks for the feedback, that will save us some time later.